rbcar_pose_topic: current_pose
rbcar_desiredpose_topic: next_goal
rbcar_cmdvel_topic: denmpc/cmd_vel

control_period: 0.1

max_speed: 1.0
max_steering_angle: 2.7
max_acceleration: 10.0

distance_offset: 0.1
theta_offset: 0.2

horizon_diskretization: 10.0
horizon_length: 10.0
